src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -63,6 +100,30 @@ const SidebarWrapper = styled('div')`
   background: #fff;
   border-right: 1px solid ${p => p.theme.borderLight};
   padding: ${space(4)};
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    display: ${p => (p.isVisible ? 'none' : 'block')};
+    animation: ${p => (p.isVisible ? 'menuOut' : 'menuIn')} 0.35s ease-in-out;
+    position: absolute;
+    z-index: 3;
```

Comment:
Is this something we should put in `theme.tsx`?

Reply:
@billyvg It might fit better in the `animations.tsx`? Maybe I could use the `slideInLeft` from that file.

Reply:
Were you talking about the `z-index` here @billyvg ?

Reply:
Yes `z-index`